content/chainguard/chainguard-images/security-advisories/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "How to Use Chainguard Security Advisories"
+linktitle: "Security Advisories"
+type: "article"
+description: "Article outlining how one can explore and use the Security Advisories found on the Chainguard Image Directory."
+date: 2023-12-27T11:07:52+02:00
+lastmod: 2023-12-27T11:07:52+02:00
+draft: false
+tags: ["Overview", "Product", "Chainguard Images", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 740
+toc: true
+---
+
+When you're checking to determine what vulnerabilities, if any, are present in a Chainguard Image, you may be using an image scanning tool like Grype or Docker Scout. Chainguard Images typically report few or zero CVEs, but when older versions of software are included in a given Image, a scanner may report more. Not all vulnerabilities are equal, and it may not be immediately clear whether a CVE found within a given Chainguard Image is likely to cause security issues in your software. Scanners can also misreport vulnerabilities found in Chainguard Images; they may return false positives that have, in reality, been fixed. 
+
+To help demystify the nature of CVEs within Chainguard Images, we've created a self-service [Security Advisories page](https://images.chainguard.dev/security) that lists every security advisory published for Chainguard Images. Having this information available allows you to view whether Chainguard is aware of a specific vulnerability reported to exist within a Chainguard Image and whether we've mitigated or are planning to mitigate the CVE.
+
+This guide outlines how you can use Chainguard's Security Advisories to learn more about the status of a CVE within a given package. It will walk through a practical example of discovering a vulnerability in a Chainguard Image, searching for Security Advisories associated with this vulnerability, and then comparing the original Image with a later version.
+
+
+## Prerequisites
+
+You don't need any special access or software to explore Chainguard's Security Advisories. However, this guide includes a few examples that use specific software tools in order to outline a practical example of how one might navigate and use the Security Advisories.
+
+To follow along with these examples, you'll need the following tools installed.
+
+* [Docker Scout](https://docs.docker.com/scout/) — A component of the Docker container platform, Docker Scout allows you to scan container images to identify potential vulnerabilities. To use Docker Scout, you will need to [install Docker](https://docs.docker.com/get-docker/) on your chosen platform.
```

Comment:
Can we change this to something like:

"A security scanner such as Trivy, Grype or Docker Scout. In the following examples we use Docker Scout...."

It's just that scout is commercial so some people won't want to use it.